sdk/agrifood/agrifood-farming-rest/package.json

```diff
@@ -95,6 +96,7 @@
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
     "@azure/identity": "^2.0.1",
     "@azure/test-utils": "^1.0.0",
+    "@azure-tools/test-credential": "~1.0.0",
     "@azure-tools/test-recorder": "^1.0.0",
```

Comment:
test-recorder needs to be updated to `^2.0.0` please follow this migration guide: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/test-utils/recorder/MIGRATION.md

fyi - @timovv

Reply:
Updated test-recorder to 2.0.0